.github/ISSUE_TEMPLATE/🐜bug-report--vs-code-runtime-extension.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "\U0001F41Cbug report: vs-code runtime extension"
```

Comment:
I love the idea of using the bug emoji! I think that this pattern of keeping track of the assignees won't be necessary as it is in the GH UI, but feel free to put that bug elsewhere 😄

Reply:
Sure😄